File: 1528-kids-with-the-greatest-number-of-candies/kids-with-the-greatest-number-of-candies.py
class Solution:
    def kidsWithCandies(self, candies: List[int], extraCandies: int) -> List[bool]:
        max_cand = max(candies)
        n = len(candies)
        ret = [False]*n
        for i in range(n):
            if max_cand <= candies[i]+extraCandies:
                ret[i] = True
        return ret

File: 1528-kids-with-the-greatest-number-of-candies/kids-with-the-greatest-number-of-candies.ts
function kidsWithCandies(candies: number[], extraCandies: number): boolean[] {
    let n = candies.length 
    let ret = Array(n).fill(false);
    let max = Math.max(...candies)// ... is spread operator?
    for (let i = 0;i<n;i++){
        if (candies[i]+extraCandies >= max){
            ret[i] = true
        }
    }  
    return ret
};